scripts/new_project.kts

```diff
@@ -1,25 +1,55 @@
+#!/usr/bin/env kscript
+
+import java.io.File
+
 object NewProject {
 
+    private const val ARGUMENT_DELIMITER = "="
     private const val KEY_APP_NAME = "app-name"
     private const val KEY_PACKAGE_NAME = "package-name"
-    private const val ARGUMENT_DELIMITER = "="
+    private const val TEMPLATE_FOLDER_NAME = "CoroutineTemplate"
 
     private var appName = ""
+    private val appNameWithoutSpace: String
+        get() = appName.trim().replace(" ", "")
```

Comment:
Oh, could be it we don't actually need `trim()`, as we are already removing all the whitespaces with  `replace(" ", "")` ? 🙏